mw-webapp/src/pages/Page404/Page404.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from "src/pages/Page404/Page404.module.scss";
+
+const ERROR_404 = "404 NOT FOUND";
```

Comment:
In this case, where string is only used in one place and is unlikely to be reused, i don't see a reason to make it a variable, little complexity without benefit

Reply:
I think using a variable here is breaking KISS.

1. String appears only once
2. String will never change
3. Meaning of the string is obvious from the context

By this logic we should make every simple html text into variables

Reply:
It's useful because this text is a magic value. All magic values should be move out in consts.

Reply:
"A "magic value" is a string or number used in a program that is essential to its proper function but provides no context or explanation for why it is what it is."

Only definition that i found, and by it - you cant really call this string a magic value, context is sufficient, will be happy to see other definition

Reply:
@Howl404 agreed with you, but it is just a convention for the current project to move all these kinds of values to variables. In my opinion, it is simpler to put all variables in constants than teach everybody to differentiate without debates about what will be used somewhere else and what is the real magic number. BTW ,  you provide good 3 points above, I like them :) You could provide clear convention for what should be placed in separate variable and what should not and we can discuss  it later